plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetFileWriterFactory.java

```diff
@@ -114,7 +114,7 @@ public Optional<FileWriter> createFileWriter(
                 return null;
             };
 
-            ParquetSchemaConverter schemaConverter = new ParquetSchemaConverter(fileColumnTypes, fileColumnNames);
+            ParquetSchemaConverter schemaConverter = new ParquetSchemaConverter(fileColumnTypes, fileColumnNames, true);
```

Comment:
This `true` is in 3 places: here, /StandardFileFormats.java and the ParquetTester.

Let's capture this important shared behavior in a constant called `HIVE_PARQUET_USE_LEGACY_DECIMAL_ENCODING`.